prod-server-integration-tests/test/admin-web-api.js

```diff
@@ -69,4 +69,46 @@ describe('admin.* Web APIs', function () {
       assert.equal(get3.no_settings_applied.length, userIds.length);
     });
   });
-});
\ No newline at end of file
+
+  describe('admin.auth.policy.{assign|get|remove}Entities', function () {
+    /* 
+      To run this test suite manually, you need an email_password auth policy
+      enabled on the Enterprise Org. For this, you need to have an Enterprise Org
+      with SSO enabled. You will additionally need to export a User ID for 
+      a user managed by the IDP for testing. 
+
+      export SLACK_SDK_TEST_GRID_USER_ID= 
+     */
+    it('should assign an entity', async function () {
+      try {
+        const res = await orgAdminClient.admin.auth.policy.assignEntities({
+          entity_ids: [process.env.SLACK_SDK_TEST_GRID_USER_ID],
+          entity_type: "USER",
+          policy_name: "email_password",
+        });
+        assert.equal(res.ok, true);
+      } catch (error) {
+        console.log(error);
```

Comment:
I can imagine the reason why you went this way (accepting the possible errors due to the data state, right?). This is fine for now 👍  Ideally, we can make these to be more like a robust scenario test but we don't need to put so much efforts for the perfection here.

Reply:
Yes, the `try catch` was helpful here in inspecting the error state (more than just the error reason), but since tests that should fail due to error show as passing, I think it's probably best to avoid false positives in our tests, so I have removed the try catch.